### PR TITLE
Adds more dungeon adjectives, nouns, and epithets

### PR DESCRIPTION
**Pull request by duncgibbs**

### CODE REVIEW

**Review by Arctem — COMMENTED** [review]
I might have a "wet" or "watery" dungeon type that would like "dripping" in the future, but otherwise I like the cave changes so far.

**Comment by Arctem** [comment]
Please sort the names within each category.

**Comment by duncgibbs** [comment]
@Arctem Sort the values alphabetically?

**Comment by Arctem** [comment]
> @Arctem Sort the values alphabetically?

Yes

File: tavern/raws/dungeon.py [hunk 1]
```diff
@@ -15,23 +15,23 @@ def __init__(self, name, adjectives=set(), nouns=set(), epithets=set(), monster_
...
+                epithets={'endless gears', 'ethereal futures', 'hypnotic rhythm', 'incessant progress', 'perpetual motion', 'ringing bells', 'the conscious cogs', 'the grandfather', 'the grinding gears', 'the twelve hands'},
                 monster_reqs={'automaton', 'golem', 'wind-up'},
                 monster_opts={'quick', 'reconstructed'},
                 ),
     DungeonType('feyland',
-                adjectives={'half-forgotten', 'ever-fading', 'mythic'},
-                nouns={'dream', 'fable'},
-                epithets={'wonders', 'hidden desires', 'the fey', 'wonders'},
+                adjectives={'auroral', 'deceptive', 'ever-fading', 'half-forgotten', 'impish', 'mythic', 'otherworldly', 'shrouded', 'twisted', 'whimsical'},
+                nouns={'court', 'dimension', 'dominion', 'dream', 'fable', 'haven', 'holding', 'kingdom', 'realm', 'vision'},
```
Comment (Arctem) [inline tavern/raws/dungeon.py:33]: All of these feel like they belong better in the "manor" section: court, dominion, holding, kingdom, realm

File: tavern/raws/dungeon.py [hunk 2]
```diff
@@ -15,100 +15,100 @@ def __init__(self, name, adjectives=set(), nouns=set(), epithets=set(), monster_
...
+                nouns={'bush', 'canopy', 'hinterland', 'jungle', 'morass', 'rainforest', 'rone', 'tangle', 'tropic', 'wilds'},
+                epithets={'acid rain', 'ceaseless rain', 'frogs', 'lizard-kings', 'the angry flood', 'the great monsoon', 'the great web', 'the invisible tigers', 'the rain queen', 'venom'},
                 monster_reqs={'spider', 'lizard', 'frogman'},
                 monster_opts={'poison', 'shaman'},
                 ),
     DungeonType('magical',
-                adjectives={'arcane', 'magic', 'occult'},
-                nouns={'nexus', 'sanctum', 'obelisk'},
-                epithets={'the arcane', 'power', 'knowledge', "wizard's rest"},
+                adjectives={'anagogic', 'arcane', 'enchanted', 'inexplicable', 'magic', 'mystic', 'occult', 'runic', 'spellbinding', 'thaumaturgic'},
```
Comment (Arctem) [inline tavern/raws/dungeon.py:60]: Maybe runic -> runed?

**Review by Arctem — APPROVED** [review]

**Pull request merged**